helpers.lua

```diff
@@ -33,6 +33,10 @@ this.max_num = function(table)
     end
     return max
 end
+this.max_n = function(note_ids, n)
+    table.sort(note_ids)
+    return { table.unpack(note_ids, math.max(#note_ids - n + 1, 1), #note_ids) }
```

Comment:
Calling`table.unpack` causes an error: `attempt to call field 'unpack' (a nil value)`. To fix the error, change `table.unpack` to `this.unpack`.

Reply:
Thank you